src/bulkExport/updateStatus.test.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as AWSMock from 'aws-sdk-mock';
+import AWS from 'aws-sdk';
+import { updateStatusStatusHandler } from './updateStatus';
+
+AWSMock.setSDKInstance(AWS);
+
+describe('updateStatus', () => {
+    beforeEach(() => {
+        process.env.GLUE_JOB_NAME = 'jobName';
+        AWSMock.restore();
+    });
+
+    test('valid status', async () => {
+        AWSMock.mock('DynamoDB', 'updateItem', (params: any, callback: Function) => {
+            callback(null);
+        });
+        await expect(
+            updateStatusStatusHandler({ jobId: '1', status: 'completed' }, null as any, null as any),
+        ).resolves.toBeUndefined();
+    });
+
+    test('invalid status', async () => {
```

Comment:
Possible to have a null status?

Reply:
null/undefined are invalid status. I just added tests for them